build/schema.js

```diff
@@ -13,21 +13,27 @@
             "properties": {
                 "alphabet": {
                     "type": "string",
-                    "minlength": 4,
-                    "maxlength": 4
+                    "pattern": "^[A-Z]{1}[a-z]{3}$"
                 },
                 "continent": {
-                    "type": "string",
-                    "minimum": 4,
-                    "maximum": 4
+                    "anyOf": [{
+                        "type": "string",
+                        "pattern": "^[A-Z]{2}$"
+                    }, {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "pattern": "^[A-Z]{2}$"
+                        }
+                    }]
                 },
                 "language": {
                     "type": "string",
-                    "minlength": 1
+                    "pattern": "^[A-Za-z]+$"
```

Comment:
If you look at the [Spanish example line 115](https://gist.github.com/Mottie/bbe593607eb0e97f3ac3de562d0a40cf#file-es-js-L115), you'll see an inverted interobang (`⸘`) which would decompose into `?!`.